connectors/es/sink.py

```diff
@@ -448,6 +455,76 @@ async def get_docs(self, generator):
 
         await self.queue.put("END_DOCS")
 
+    async def get_docs_incrementally(self, generator):
```

Comment:
Want to make it isolated from `get_docs` (full sync).

Once we use Delete by query API, the `get_docs` method can be deleted, and this method will be renamed to `get_docs`

Reply:
Can you please create an issue to make sure we won't forget about this?

Reply:
Added: https://github.com/elastic/enterprise-search-team/issues/4712